src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
Ok, in meantime package version changed to 2.0.10, that is fixed now. But, until this current PR/branch is not picked up by the feed, the Win81 Sample will fail at runtime, due to a breaking change introduced by this PR.